src/core/oauth2-authorize.js

```diff
@@ -66,6 +68,22 @@ export default function authorize ( { auth, authActions, errActions, configs, au
     query.push("realm=" + encodeURIComponent(authConfigs.realm))
   }
 
+  if (isCode && authConfigs.usePkceWithAuthorizationCodeGrant) {
```

Comment:
**Change request**: instead of setting and using `isCode`, let's use `flow === "authorizationCode"`, unless there's a reason not to that I'm missing.

Reply:
No particular reason at all. I thought I would reuse the fact this condition has been already evaluated. I changed it to your suggestion.